pages/newsletter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { NextPage } from "next";
+import Image from "next/image";
+import styles from "../styles/components/newsletter.module.css";
+import Button from "../components/UI/button";
+import { useState } from "react";
+import { useAccount } from "@starknet-react/core";
+
+const NewsletterPage: NextPage = () => {
+  const { address } = useAccount();
+  const [error, setError] = useState<string>("");
+  const submitHandler = () => {
+    const email = (document.getElementById("email") as HTMLInputElement).value;
+    if (!email) {
+      setError("Please enter your email address.");
+      return;
+    }
+    fetch(`${process.env.NEXT_PUBLIC_SALES_SERVER_LINK}/newsletter_subscribe`, {
```

Comment:
Did you test that this back-end also triggers an add in the Mailerlite database ?